ci/jwt/jwks_server.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+
+import argparse
+import base64
+import json
+import logging
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from collections import OrderedDict
+from jwcrypto import jwt, jwk
+
+DEFAULT_PORT = 8080
+
+keys = {}
```

Comment:
Don't use keys as global. Move it between the functions

Reply:
Between which functions? Separate instance of the `JWKSRequestHandler` class is created for each request. It means that global state can not be in this class.

Reply:
Can it be in outside singalton class ? With function make the accessing to it easier?
If not leave it as it is

Reply:
I sure that there's more elegant way to do it but I'm doubting is it worth ROI...